modules/ROOT/pages/api-groups-sla-tiers.adoc

```diff
@@ -0,0 +1,53 @@
+= SLA Tiers For API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+As a Group Administrator, you can specify service-level agreement (SLA) tiers for API Groups instances to limit the number of requests an application (user) can make to the APIs in an API Group. SLAs enable you to protect your API Group instances and help manage network traffic to your APIs.
+
+The SLA tier definition also determines whether access to the API Groups instance requires your approval. You can configure manual or automatic approvals to requests.
+
+== SLA Types
```

Comment:
Is this the best title? This text is not about types so much as the limits on types.

Reply:
This topic should be reorganized so that the sections are chronological and lead narratively to the next. Later.

Reply:
Page to be removed. Done! Thanks!